hack/graph-util.py

```diff
@@ -30,6 +30,13 @@
 _LOGGER = logging.getLogger(__name__)
 _LOGGER.setLevel(logging.DEBUG)
 
+def get_architecture(meta):
+    return meta['image-config-data']['architecture']
```

Comment:
```suggestion
    return meta.get('image-config-data', {}).get('architecture')
```
In case `meta` doesn't have these

Reply:
Although I'm not sure - do we still have releases without arch info? @wking or @steveeJ ?

Reply:
> ...do we still have releases without arch info...

Arch info is extracted from the container image itself; there's no way you can avoid having it.  Sanity check:

```console
$ hack/graph-util.py push-to-quay
DEBUG: caching metadata for 4.2.18 quay.io/openshift-release-dev/ocp-release@sha256:144485daf08522e7fab77cb90ce360e72aca7a92f633a74b522808b4fe2b6627
...
$ grep -chr architecture .nodes | sort | uniq -c
      1 0
    100 1
$ grep -cr architecture .nodes | grep ':0$'
.nodes/version:0
```

(i.e. "the only file under `.nodes` without an `architecture` is the cache-version file").  So I'm fine with this as @eparis has it in 0f777c88b.

Reply:
Although personally I would not have bothered wrapping this one-liner in a `get_architecture` function, but whatever ;)

Reply:
hah, and I see @vrutkovs explicitly asking for a `get_architecture` wrapper [here][1].

[1]: https://github.com/openshift/cincinnati-graph-data/pull/42#discussion_r374623634

Reply:
Ah, great, lets keep it like this then